docs/sources/datasources/influxdb/configure-influxdb-data-source/_index.md

```diff
@@ -0,0 +1,260 @@
+- **Client cert** - Add the client certificate.
+- **Client key** - Add the client key.
+
+**Custom HTTP headers:**
+
+- **Header** - Add a custom HTTP header. Select an option from the drop-down. Allows custom headers to be passed based on the needs of your InfluxDB instance.
+- **Value** - The value for the header.
+
+**Private data source connect:**
+
+- **Private data source connect** - _Only for Grafana Cloud users._ Private data source connect, or PDC, allows you to establish a private, secured connection between a Grafana Cloud instance, or stack, and data sources secured within a private network. Click the drop-down to locate the URL for PDC. For more information regarding Grafana PDC refer to [Private data source connect (PDC)](https://grafana.com/docs/grafana-cloud/connect-externally-hosted/private-data-source-connect/).
+
+Click **Manage private data source connect** to be taken to your PDC connection page, where you'll find your PDC configuration details.
+
+Once you have added your connection settings, click **Save & test** to test the data source connection.
+
+### InfluxQL-specific configuration section
+
+The following settings are specific to the InfluxQL query language option.
+
+**InfluxQL InfluxDB details section:**
+
+- **Database** - Sets the ID of the bucket to query. Refer to [View buckets](https://docs.influxdata.com/influxdb/v2.0/organizations/buckets/view-buckets/) in InfluxData's documentation on how to locate the list of available buckets and their corresponding IDs.
+- **User** - The user name used to sign in to InfluxDB.
+- **Password** - Defines the token used to query the bucket defined in **Database**. Retrieve the password from the [Tokens page](https://docs.influxdata.com/influxdb/v2.0/security/tokens/view-tokens/) of the InfluxDB UI.
+- **HTTP method** - Sets the HTTP method used to query your data source. The POST method allows for larger queries that would return an error using the GET method. The default method is `POST`.
+- **Min time interval** - _(Optional)_ Sets the minimum time interval for auto group-by.  Grafana recommends  setting this to match the data write frequency. For example, if your data is written every minute, it’s recommended to set this interval to 1 minute, so that each group contains data from each new write. The default is `10s`. Refer to [Min time interval](#configure-min-time-interval) for format examples.
+- **Max series** - _(Optional)_ Sets a limit on the maximum number of series or tables that Grafana processes.  Set a lower limit to prevent system overload, or increase it if you have many small time series and need to display more of them. The default is `1000`.
+
+### SQL-specific configuration section
+
+The following settings are specific to the SQL query language option.
+
+**SQL InfluxDB details section:**
+
+- **Database** - Add the ID of the bucket to query. Obtain this from the **Buckets page** of the InfluxDB UI. 
```

Comment:
Nit: Specify the bucket ID. See the **Buckets page** in the InfluxDB UI to locate the ID.

Reply:
fixed